miqsel/miqsel.py

```diff
@@ -167,8 +175,10 @@ def start(ctx, connection):
     container = get_container()
     img = connection.conf.get("image")
     name = connection.conf.get("container_name")
-
+    
     if not container:
+        if not get_image():
+            print("Pulling docker images...Please wait...It will take some time!")
```

Comment:
Required:
1. I would use `click.echo()` than `print()`
2. first message should
`click.echo("Pulling docker image...")`
3. second message should
`click.echo("It will take time; please wait for some some time...")`